.github/workflows/test.yml

```diff
@@ -36,15 +36,13 @@ jobs:
         ports:
           - 5432:5432
     steps:
-      - uses: actions/checkout@v2
```

Comment:
I still see a lot of `actions/cache@v2` in the code, can we make sure we either change all of them or none of them, just to keep it consistent